backend/blueprint/api/app/__init__.py

```diff
@@ -0,0 +1,20 @@
+import os
```

Comment:
Why to move in a different folder?

Reply:
Ok, you are right, I am correcting this. Thanks for pointing it out